ros2/source/source/Dockerfile

```
@@ -72,17 +27,20 @@ RUN apt-get update && rosdep install -y \
     && rm -rf /var/lib/apt/lists/*
 
 # build source
-WORKDIR $ROS2_WS
 RUN colcon \
     build \
     --cmake-args -DSECURITY=ON --no-warn-unused-cli \
```

Could we tabulate these so colcon build args are revisioned on separate lines for VCS?
```suggestion
     --cmake-args \
       -DSECURITY=ON \
     --no-warn-unused-cli \
```

---

This comes from the images.yaml.em file. This PR doesn't modify the logic surrounding colcon args expansion.
Trying to keep the scope here minimal I was not planning on addressing this in this PR.

---

:lipstick: later then.